docs/moments.rst

```diff
@@ -63,25 +63,15 @@ plotting packages such as APLpy::
     >>> f.save('moment_0.png')  # doctest: +SKIP
 
 The equation for the N-th moment, where N is usually an integer between -1 and some arbitrary
-positive integer say 11, is as follows:
+positive integer, is as follows:
 
 .. math:: M_N = \frac{\int I (l - M_l)^N dl}{M_0}
 
-Descriptions for the twelve most common moments used are as follows:
-
-moments=-1 - the mean value of the spectrum  
-moments=0  - the integrated value of the spectrum, can be used for computing line ratios
-moments=1  - the intensity weighted coordinate; by convention used to get the ’velocity fields’
-moments=2  - the intensity weighted dispersion of the coordinate; by convention used to get the ’velocity dispersion’  
-moments=3  - the median of the cube I
-moments=4  - the median coordinate  
-moments=5  - the standard deviation (SD) about the mean of the spectrum  
-moments=6  - the root mean square of the spectrum  
-moments=7  - the absolute mean deviation of the spectrum  
-moments=8  - the maximum value of the spectrum  
-moments=9  - the coordinate of the maximum value of the spectrum  
-moments=10 - the minimum value of the spectrum  
-moments=11 - the coordinate of the minimum value of the spectrum
+Descriptions for the three most common moments used are as follows:
+
+* 0th moment - the integrated flux over the spectral line, can be used for computing some line ratios
```

Comment:
probably better to say "integrated intensity" than "integrated flux" - usually radio cubes are in intensity units.

Instead of ", can be used", say something like, "The integrated intensity is often used in computing line ratios."

Reply:
Okay, understood.